x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -116,11 +123,11 @@ export function AlertsPage({ routeParams }: AlertsPageProps) {
           </>
         ),
         rightSideItems: [
-          <EuiButton fill href={manageDetectionRulesHref} iconType="gear">
-            {i18n.translate('xpack.observability.alerts.manageDetectionRulesButtonLabel', {
-              defaultMessage: 'Manage detection rules',
+          <EuiButtonEmpty href={manageDetectionRulesHref}>
```

Comment:
Thinking maybe we should update the const to `manageRulesHref` to keep the naming aligned. Doesn't look like anything else uses the `manageDetectionRulesHref` name.

Reply:
@matschaffer Yep good point! Will change that